credentials/java/com/google/auth/Credentials.java

```diff
@@ -26,6 +26,8 @@
    * populated in headers or other context. The operation can block and fail to complete and may do
    * things such as refreshing access tokens.
    *
+   * <p>If a key ends with {@code "-bin"}, its values are base64-encoded binary data.
+   *
    * @throws IOException if there was an error getting up-to-date access.
```

Comment:
Perhaps. "The convention for handling binary data is for the key in returned map to end with "-bin" and for the corresponding values to be base64 encoded."